spec/features/brexit_checker/question_navigation_spec.rb

```diff
@@ -0,0 +1,42 @@
+require "spec_helper"
+
+RSpec.feature "Navigating Brexit Checker questions", type: :feature do
+  scenario "navigate to previous question" do
```

Comment:
Just a minor thing: we conventionally drop the scenario text if there's only one scenario.